src/module/actor/sheets/SR5BaseActorSheet.ts

```diff
@@ -371,10 +373,46 @@ export class SR5BaseActorSheet extends foundry.appv1.sheets.ActorSheet {
         // Reset Actor Run Data
         html.find('.reset-actor-run-data').on('click', this._onResetActorRunData.bind(this));
 
-        // Matrix Network
-        html.find('.connect-to-network').on('click', this._onConnectToMatrixNetwork.bind(this));
-        // Matrix Target - Connected Icons Visibility Switch
-        html.find('.toggle-connected-matrix-icons').on('click', this._onToggleConnectedMatrixIcons.bind(this));
+        html.find('select[name="initiative-select"]').on('change', this._onInitiativePerceptionChange.bind(this));
+    }
+
+    /**
+     * Get the options for Initiative Perception
+     */
+    _prepareInitiativePerception() {
+        const initiative = this.actor.system.initiative.perception;
+        if (initiative === 'matrix') {
+            return this.actor.isUsingHotSim ? 'hot_sim' : 'cold_sim';
+        }
+        return initiative;
+    }
+
+    /**
+     * Handle Changing Initiative Perception
+     * - the select handles hot sim vs cold sim and doesn't match our dataset exactly
+     * - this is more of a band-aid until we do appv2
+     * @param event
+     */
+    async _onInitiativePerceptionChange(event) {
+        const newValue = event.currentTarget?.value;
+        if (newValue === 'meatspace' || newValue === 'magic') {
+            // meatspace and magic can be directly applied as the perception type
+            // disable VR as well
+            await this.actor.update({ system: {
+                    initiative: { perception: newValue, },
+                    matrix: { vr: false }
+                }});
+        } else if (newValue === 'hot_sim' || newValue === 'cold_sim') {
```

Comment:
Shouldn´t we be able to check for `actor.system.matrix.vr` as well or is it not fully supported yet by setting sim modes?

Reply:
The intention here is to have the Initiative Select control the VR status as well. I removed the ColdSim/HotSim/AugmentedReality in the matrix tab in favor of just using this select. 

This is mostly a hold-over until we do appv2 which will probably introduce a better way to select the "Presence" of the actor.